### ISSUE TITLE
Metrics improvements

### ISSUE THREAD

**Issue by Disper** [kyma-project/infrastructure-manager#180]
**Description**
- [ ] **(Most important)** enable `Instant` options for existing graphs for KIM metrics to speed up how fast the dashboards are loaded and limit the chances that too big load will be generated for Victoria metrics
- [ ] Consider creation of a counter that would depict how many clusters are in the "gray area" meaning that according to the configuration the rotation should already happen while it did not. This could be also a graph that would show got this gray area (sum of such clusters) is changing over time.
- [ ] usability of plutono dashboard would increase if it would be possible to choose a selector (like `shootName`) and narrow the graphs and tables to metrics only concerning that single cluster 
- [ ] renaming metrics so they will be following the [prometheus best practices](https://prometheus.io/docs/practices/naming/) to increase the ease of maintenance 

**Attachments**
- [prometheus best practices](https://prometheus.io/docs/practices/naming/)

**Comment by tobiscr** [comment]
Relates to #113

**Comment by tobiscr** [comment]
@Disper : do we still need this issue?

**Comment by Disper** [comment]
Yes, the metrics related to `kubeconfig rotation` were not changed with the latest addition of `Runtime CR` metrics
